### PR TITLE
Add wrappers for dashboards and insights

### PR DESCRIPTION
**Pull request by adescamps**
[sc-95092]
[sc-95094]

### CODE REVIEW

File: dataikuapi/dss/dashboard.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+class DSSDashboard(object):
+    """
+    A handle to interact with a dashboard on the DSS instance.
+    Do not create this directly, use :meth:`dataikuapi.dss.DSSProject.get_scenario`
+    """
+    def __init__(self, client, project_key, dashboard_id):
+        """
+        :param DSSClient client: an api client to connect to the DSS backend
+        :param str project_key: identifier of the project to access
+        :param str dashboard_id: identified of the dashboard
```
Comment (joubert-m) [inline dataikuapi/dss/dashboard.py:26]:
```suggestion
        :param str dashboard_id: identifier of the dashboard
```

File: dataikuapi/dss/dashboard.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        return DSSDashboardSettings(self.client, self.client._perform_json("GET", DASHBOARD_URI_FORMAT % (self.project_key, self.dashboard_id)))
+
+class DSSDashboardListItem(DSSTaggableObjectListItem):
+    """An item in a list of dashboards. Do not instantiate this class, use :meth:`dataikuapi.dss.project.DSSProject.list_dashboards"""
+    def __init__(self, client, data):
+        super(DSSDashboardListItem, self).__init__(data)
+        self.client = client
+
+    def to_dashboard(self):
+        return DSSDashboard(self.client, self._data["projectKey"], self._data["id"])
```
Comment (joubert-m) [inline dataikuapi/dss/dashboard.py:54]: we sometimes use " and sometimes ' for string we should harmonise it a bit

File: dataikuapi/dss/insight.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+class DSSInsight(object):
+    """
+    A handle to interact with an insight on the DSS instance.
+    Do not create this directly, use :meth:`dataikuapi.dss.DSSProject.get_insight`
+    """
+    def __init__(self, client, project_key, insight_id):
+        """
+        :param DSSClient client: an api client to connect to the DSS backend
+        :param str project_key: identifier of the project to access
+        :param str insight_id: identified of the insight
```
Comment (joubert-m) [inline dataikuapi/dss/insight.py:26]:
```suggestion
        :param str insight_id: identifier of the insight
```

**Review by joubert-m — COMMENTED** [review]
some nitpicking otherwise LGTM

**Review by geraldlamory — COMMENTED** [review]
Tested manually, it works well. And the code looks good 👍 

Maybe we could add an argument to `list_insights` to retrieve insights as a list of `DSSInsightListItem` like for dashboards ?

**Pull request closed without merging**